docs/messaging/messaging-spans.md

```diff
@@ -153,38 +153,32 @@ in such a way that it cannot be changed by intermediaries.
 
-```
-<destination name> <operation name>
-```
+<!-- markdown-link-check-disable -->
+<!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
+The **span name** SHOULD be `{messaging.operation.name} {destination}` (see below for the exact definition of the [`{destination}`](#destination-placeholder) placeholder).
+<!-- markdown-link-check-enable -->
 
-If the operation name is not specified by the messaging system, then the operation type as defined in [Operation types](#operation-types) SHOULD be used:
+Semantic conventions for individual messaging systems MAY specify different span name format and then MUST document it in [semantic conventions for specific messaging technologies](#semantic-conventions-for-specific-messaging-technologies).
 
-```
-<destination name> <operation type>
-```
+The <span id="destination-placeholder">`{destination}`</span> SHOULD adhere to one of the following values, arranged in prioritized order, provided they are accessible:
 
-The destination name SHOULD only be used for the span name if it is known to be of low cardinality (cf. [general span name guidelines](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/trace/api.md#span)).
-This can be assumed if it is statically derived from application code or configuration.
-Wherever possible, the real destination names after resolving logical or aliased names SHOULD be used.
-If the destination name is dynamic, such as a [conversation ID](#conversations) or a value obtained from a `Reply-To` header, it SHOULD NOT be used for the span name.
-In these cases, an artificial destination name that best expresses the destination, or a generic, static fallback like `"(anonymous)"` for [anonymous destinations](#temporary-and-anonymous-destinations) SHOULD be used instead.
+- `messaging.destination.template`
+- `messaging.destination.name` if it is known to be of a low cardinality.
+- `server.address:server.port`
+- `messaging.system`
 
 Examples:
 
-* `shop.orders publish`
-* `shop.orders subscribe`
-* `shop.orders settle`
-* `print_jobs publish`
-* `print_jobs nack`
-* `topic with spaces process`
-* `AuthenticationRequest-Conversations settle`
-* `(anonymous) send` (`(anonymous)` being a stable identifier for an unnamed destination)
-
-Messaging system specific adaptions to span naming MUST be documented in [semantic conventions for specific messaging technologies](#semantic-conventions-for-specific-messaging-technologies).
+* `publish shop.orders`
+* `subscribe shop.orders`
+* `ack shop.orders`
+* `nack print_jobs `
+* `process topic with spaces `
+* `settle AuthenticationRequest-Conversations`
 
-### Operation types
+### Common messaging operations
 
 The following operation types related to messages are defined for these semantic conventions:
 
```

Comment:
As we remove the operation type attribute, we shouldn't write about "operation types" anymore but only about "operation names".

Also we'd need to clarify the meaning of the table above. Would we recommend using those well-known names (if applicable)? Should we require systems to document other names that are used and not in this list? Do we always require the operation name "process" for processing operations?

Reply:
whether we remove it or not, we still need to establish terminology for the rest of the document. If we remove the attribute, we can add a note that it's the only goal and that type does not appear on the telemetry.

Reply:
resolving since we're keeping the operation type